### ISSUE TITLE
Complete testing code

### ISSUE THREAD

**Issue by Emma1066** [nju-websoft/FAN#5]
Hi, thank you for sharing your work.

I saw you uploaded the testing code in the early issue. Could you please help me with the two problems regarding the testing code?  
1. Among the input of the `evaluate(model, data_loader, config)` function, is `model` the loaded 'fan.pkl' ？ How to construct `dataloader` ?
2. Where can I find `get_inputs()` function and `model.get_loss()` function ?
~~~      
   for raw_batch in bar:
            inputs, labels = get_inputs(raw_batch, config)
            logits = model(**inputs)
            loss = model.get_loss(logits, labels)
~~~

Thank you very much.
